src/test/java/uk/gov/hmcts/juror/api/moj/report/DataTypeTest.java

```diff
@@ -181,6 +181,12 @@ void poolNumber() {
             String.class, QPoolRequest.poolRequest.poolNumber, QPoolRequest.poolRequest);
     }
 
+    @Test
+    void jurorAddress() {
+        assertMatchesCombined(DataType.JUROR_ADDRESS, "juror_address", "Address", List.class,
+            DataType.ADDRESS_LINE_1, DataType.ADDRESS_LINE_2, DataType.ADDRESS_LINE_3, DataType.ADDRESS_LINE_4,
+            DataType.ADDRESS_LINE_5, DataType.POSTCODE);
+    }
```

Comment:
Need to test every new DataType entry 

- ADDRESS_LINE_1
- ADDRESS_LINE_2
- ADDRESS_LINE_3
- ADDRESS_LINE_4
- ADDRESS_LINE_5